docs/output.md

```diff
@@ -53,6 +54,46 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
 
 Results generated by MultiQC collate pipeline QC from supported tools e.g. FastQC. The pipeline has special steps which also allow the software versions to be reported in the MultiQC output for future traceability. For more information about how to use MultiQC reports, see <http://multiqc.info>.
 
+### Picard
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `picard/`
+  - `*.CollectHsMetrics.coverage_metrics`: a standalone file that contains hybrid-selection metrics for BAM files.
+  - `*.CollectMultipleMetrics.alignment_summary_metrics`: a standalone files containing alignment summary metrics from BAM files.
+  - `*.CollectMultipleMetrics.base_distribution_by_cycle.pdf`: a standalone file exhibiting summary metrics of base distributions as a pdf plot.
+  - `*.CollectMultipleMetrics.base_distribution_by_cycle_metrics`: a standalone file containing base distribution by cycles summary metrics from BAM files.
+  - `*.CollectMultipleMetrics.insert_size_histogram.pdf`: a standalone file exhibiting summary metrics of insert sizes as a pdf plot.
+  - `*.CollectMultipleMetrics.insert_size_metrics`: a standalone file containing metrics on insert sizes from BAM files.
+  - `*. CollectMultipleMetrics.quality_by_cycle.pdf`: a standalone file exhibiting summary metrics of cycle quality as a pdf plot.
+  - `*.CollectMultipleMetrics.quality_by_cycle_metrics`: a standalone file containing metrics on cycle quality from BAM files.
+  - `*.CollectMultipleMetrics.quality_distribution.pdf`: a standalone file exhibiting summary metrics of the quality distribution as a pdf plot.
+  - `*.CollectMultipleMetrics.quality_distribution_metrics`: a standalone file containing metrics on the quality distriburion of BAM files.
+  - `*.CollectMultipleMetrics.read_length_histogram.pdf`: a standalone file exhibiting summary metrics of sample read lengths as a histogram.
+  - `genome.dict`: a standalone file containing a sequence dictionary for a reference sequence.
+  - `genome.fa`: ?
+  - `genome.fa.fai`: ?
```

Comment:
Correcting this here, since this seems to be just the input.

```suggestion
  - `genome.fa`: The reference genome FASTA file used to generate the mapping alignment
  - `genome.fa.fai`: The FASTA index file for the reference genome.?
```